observatory-dags/observatory/dags/telescopes/onix.py

```diff
@@ -138,24 +83,15 @@ def transform_bucket(self):
         """
         return self.organisation.gcp_transform_bucket
 
-    def move_files_to_in_progress(self):
```

Comment:
I would prefer to leave some of these functions in here, e.g. `move_files_to_finished`, `download` (with the interface that it had) and `move_files_to_finished` (these are the interface that the telescope uses) and use the SftpFolder class from within the OnixRelease object.

Reply:
This should be resolved in the commit I just made, unless I misunderstood.